include/tf-psa-crypto/version.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \file tf-psa-crypto/version.h
```

Comment:
This does not seem to be enough to appease `check_doxygen_warning()`. Looking at `build_info.h` and `check_config.h` it is likely that in `include/mbedtls/version.h` we should have:
```
 * \file mbedtls/version.h
```